src/main/java/org/openrewrite/maven/ConfigurableRewriteMojo.java

```diff
@@ -278,6 +292,18 @@ protected List<NamedStyles> loadStyles(MavenProject project, Environment env) {
         return styles;
     }
 
+    private String toCheckStyleDocument(final Xpp3Dom dom) throws IOException {
+        StringWriter stringWriter = new StringWriter();
+
+        MXSerializer serializer = new MXSerializer();
```

Comment:
This was what I came up with using the xml classes within the Plexus libraries. Wish it could be a bit cleaner.